examples/basic-webpack/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
There should be two separate configs for development and production. It’s a common enough misconception that you can use one config for both, and people blame React for being slow.

Please see a sensible Webpack configuration here: https://github.com/MsUzoAgu/ulonkaFrontend01/pull/1/files